tests/test_flair_md.py

```diff
@@ -3,20 +3,28 @@
 
 from pathlib import Path
 
+import os
+
 from flair.models import SequenceTagger
 
 from REL.mention_detection import MentionDetection
 
 
 def test_md():
     # return standard Flair tagger + mention detection object
-    tagger = SequenceTagger.load("ner-fast")
-    md = MentionDetection(Path(__file__).parent, "wiki_test")
+    tagger_ner_name = "flair"
+    tagger_ner = SequenceTagger.load("ner-fast")
+    process_sentences = True
+    split_docs_value = 0
+    base_url = os.path.dirname(__file__) + "/../data/"
```

Comment:
Would it be possible not to hardcode this?

Reply:
Adopted: removed hard-coded paths